src/main/proto/marathon.proto

```diff
@@ -148,11 +149,42 @@ message ExtendedContainerInfo {
   }
 
   required mesos.ContainerInfo.Type type = 1;
-  repeated mesos.Volume volumes = 2;
+  repeated Volume volumes = 2;
```

Comment:
Have you tested to read volume information from older Marathon versions?
Do we need a migration? (Not sure how protobuf handles different types)